### PR TITLE
OCPBUGS-34227: fix: actually pass through and validate chunk size

### PR DESCRIPTION
**Pull request by jakobmoellerdev**

### CODE REVIEW

File: internal/controllers/vgmanager/controller.go [hunk 1]
```diff
@@ -579,6 +582,19 @@ func (r *Reconciler) validateLVs(ctx context.Context, volumeGroup *lvmv1alpha1.L
 	return nil
 }
 
+func (r *Reconciler) verifyChunkSizeForPolicy(config *lvmv1alpha1.ThinPoolConfig, lv lvm.LogicalVolume) error {
+	if config.ChunkSizeCalculationPolicy == lvmv1alpha1.ChunkSizeCalculationPolicyStatic {
```
[the remark this answers is not in the document]
Comment (jakobmoellerdev) [reply internal/controllers/vgmanager/controller.go:586]: works well for me, let me change :)

**Review by suleymanakbas91 — COMMENTED** [review]
/lgtm

**Pull request merged**